client/app/src/main/java/com/microsoft/research/karya/ui/scenarios/common/MicrotaskRenderer.kt

```diff
@@ -172,7 +204,7 @@ abstract class MicrotaskRenderer(
 
   /** Get Microtask input directory */
   private fun getMicrotaskInputDirectory(): String {
-    val microtaskInputName = KaryaFileContainer.MICROTASK_INPUT.cname
+    val microtaskInputName = BaseActivity.KaryaFileContainer.MICROTASK_INPUT.cname
```

Comment:
Can we remove the `BaseActivity` dependency from here?

Reply:
Yeah, good catch I missed it. KaryaFileContainer is already free of BaseActivity I will add that here